recipes/r-openssl/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.9.6' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-openssl
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: openssl_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/openssl_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/openssl/openssl_{{ version }}.tar.gz
+  sha256: 6dd6d1cade4004962d516ad761fff0812beec0232318b385d286761423a5dc39
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+    - openssl
+
+  run:
+    - r-base
+    - openssl
```

Comment:
Good point. In fact, it is not always like that. Sometimes, libgcc is not used. But I agree, it is safer to always add it. I will update the skeleton when I find the time.